Themis.xcodeproj/xcshareddata/xcschemes/Themis.xcscheme

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file shouldn't have been committed as the password of the test server account are exposed again :) 
@huber-florian it's time to get to work again 🙈

Reply:
No, as far as I see it is not further on develop. So please take care to rebase on develop and squash the commits so that it is also not further part of this feature branch.

Reply:
Its still in the same commit that you deleted @huber-florian today, but as the brach is older it still has the old commits...